weedcoco/importers/deposit.py

```diff
@@ -0,0 +1,107 @@
+import argparse
```

Comment:
Deposit is not an `importer`. Maybe it should be `weedcoco/repo/deposit.py` since it is now about a repository rather than the schema.